orangecontrib/text/widgets/icons/Twitter.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The color of this icon does not quite match the color of others. I there a touch of blue in this icon? I think that general rule is to stick to black, white and shades of gray. Could we change this to be more consistent?